core/src/main/java/io/codiga/analyzer/treesitterpm/TreeSitterPatternMatching.java

```diff
@@ -7,7 +7,11 @@
 import ai.serenade.treesitter.Node;
 import ai.serenade.treesitter.Parser;
 import ai.serenade.treesitter.Tree;
-import ai.serenade.treesitter.query.*;
+import ai.serenade.treesitter.query.Query;
+import ai.serenade.treesitter.query.QueryCapture;
+import ai.serenade.treesitter.query.QueryCursor;
+import ai.serenade.treesitter.query.QueryMatch;
+import ai.serenade.treesitter.query.QueryMatchCapture;
```

Comment:
Why are these imports being expanded?

Reply:
Done by IntelliJ. I did not enable the java formatter for this diff so that you can clearly see the differences. IntelliJ still added some import expansions.